plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -310,12 +312,17 @@ export function wrapFields(
   Object.keys(fields).forEach(key => {
     const field = fields[key];
 
-    if (!field) {
+    if (!field || ignoreMap().has(key)) {
```

Comment:
If I understand the flow correctly, then this will not only ignore this field, but also all the schema resolver field below it, e.g.
```
{
  foo {
    bar
  }
}
```

Then ignoring only `foo` will terminate this recursion function, preventing it from calling `wrapFields` below on `foo`s field resolver, including `bar`.

This will apply for resolvers on the schema, but for "fieldResolver" invocations, the behavior will be different, as nothing will prevent it from being called on `bar` and thus the span will be recorded, right?

I suggest refactoring the code to have consistency between the 2 resolver options, and adding a test for it.